data/source.py

```diff
@@ -0,0 +1,20 @@
+import csv
+
+from questions import ExampleQuestion
+
+
+def get_from_csv_file(path: str) -> [ExampleQuestion]:
+    with open(path, 'r', encoding='utf-8') as f:
+        return from_csv(f)
+
+
+def from_csv(iterable: any) -> [ExampleQuestion]:
```

Comment:
any type is not good, does Iterable usable?

Reply:
document of csv reader.
```
The "iterable" argument can be any object that returns a line
    of input for each iteration, such as a file object or a list.  The
    optional "dialect" parameter is discussed below.  The function
    also accepts optional keyword arguments which override settings
    provided by the dialect.
```